app/services/articles/create_with_video.rb

```diff
@@ -0,0 +1,48 @@
+module Articles
+  class CreateWithVideo
```

Comment:
I decided to move this from `ArticleWithVideoCreationService` to something that is more in line with our current conventions. In the process I also added a `call` class method, so we went from 

```ruby
ArticleWithVideoCreationService.new(article_params, current_user).create!
```

to

```ruby
Articles::CreateWithVideo.call(params[:key], current_user)
````